bugwarrior/notifications.py

```diff
@@ -91,6 +93,30 @@ def send_notification(issue, op, conf):
             priority=1,
         )
         return
+    elif notify_backend == 'applescript':
+        if op == 'bw_finished':
+            description = 'Finished querying for new issues.\n{}'.format(
+                issue['description']
+            )
+            subprocess.call(
+                [
+                    'osascript',
+                    '-e',
+                    'display notification "{}" with title "Bugwarrior"'.format(
+                        description
+                    ),
+                ]
+            )
+            return
+
+        subprocess.call(
+            [
+                'osascript',
+                '-e',
+                'display notification "{}" with title "Bugwarrior"'.format(message),
+            ]
+        )
+        return
```

Comment:
Since this implementation is simpler than growlnotify, the logic could be simpler too. E.g.:

```py
description = ...
notification = description if op == 'bw_finished' else message
subprocess.call(...
    "...".format(
        notification
...
```

Reply:
Good catch, thanks.